libs-scala/contextualized-logging/src/main/scala/com/digitalasset/logging/LoggingContext.scala

```diff
@@ -9,22 +9,23 @@ import org.slf4j.Marker
 
 import scala.collection.JavaConverters.mapAsJavaMapConverter
 
+@SuppressWarnings(Array("org.wartremover.warts.Any"))
```

Comment:
It would be great if we could restrict the scope of this warning suppression (and possibly explicitly annotate methods and calls explicitly to make it go away completely).

Reply:
Reverted to just using `String`; we'll revisit this at some point, I'm sure.